keyboards/maartenwut/atom47/rev4/rev4.h

```diff
@@ -0,0 +1,37 @@
+#ifndef ATOM47_REV4_H
+#define ATOM47_REV4_H
+
```

Comment:
```suggestion
#pragma once
```

Reply:
Why not for the other revisions?

Reply:
I can't add suggestions on unchanged files/hunks.

Reply:
So I should do it on the other revisions as well?

Reply:
That would probably be a good idea, yeah.